pages/radar.md

```diff
@@ -7,6 +7,9 @@ permalink: /radar.html
 
 [/home](/)
 
+## [Java 17 released](https://www.oracle.com/news/announcement/oracle-releases-java-17-2021-09-14/)
+14-September-2021: Java 17 is available. Moreover, the [new license is free](https://blogs.oracle.com/java/post/free-java-license), and you can [download JDK](https://www.oracle.com/java/technologies/downloads/) without registration. Java 17 contains several [new features](https://www.oracle.com/java/technologies/javase/17all-relnotes.html) (both new syntax constructs and platform updates), but we believe that the main reason users will migrate to the new version is that the virtual machine [is faster](https://www.optaplanner.org/blog/2021/09/15/HowMuchFasterIsJava17.html) than previous versions. It means you can run your application 8.66% faster than using Java 11 and 2.41% faster than using Java 16, without any effort and payments. This may also affect other languages that are compiled into Java source code or JVM bytecode. We expect that due to the combined advantages of the new version the migration to it will be quite massive and fast.
```

Comment:
14-September-2021: Java 17 is available. Moreover, the [new license is free](https://blogs.oracle.com/java/post/free-java-license), and you can [download JDK](https://www.oracle.com/java/technologies/downloads/) without registration. Java 17 contains several [new features](https://www.oracle.com/java/technologies/javase/17all-relnotes.html), including new syntax constructs and platform updates. However, we believe users will migrate to the new version mainly because the virtual machine [is faster](https://www.optaplanner.org/blog/2021/09/15/HowMuchFasterIsJava17.html) than previous versions, enabling you to run an application 8.66% faster than using Java 11 and 2.41% faster than using Java 16, without any effort and payments. It may also affect other languages that are compiled into Java source code or JVM bytecode. We expect that due to the combined advantages of the new version, the migration to it will be pretty massive and fast.